src/exo7 - Collections/exo7.exercise.ts

```diff
@@ -136,13 +167,24 @@ export const odds = new Set([1, 3, 5, 7, 9]);
 //
 // HINT:
 // - Be mindful of the order of operands for the operator you will choose.
+const isInPrimesSet = (entry: number) => {
+ return primes.has(entry)
+}
 
-export const nonPrimeOdds: ReadonlySet<number> = unimplemented();
+export const nonPrimeOdds: ReadonlySet<number> = 
+  pipe (
+    odds,
+    readonlySet.filter(entry => !isInPrimesSet(entry))
+  );
```

Comment:
This works, but you could also try to use `readonlyArraySet.difference` to write another solution 
:)